### ISSUE TITLE
Suffrage Voter Campaign

### ISSUE THREAD

**Issue by sebastianebarb** [CityOfBoston/design-team-workflow#1468]
The suffrage project is moving forward with a digital billboard campaign whose tagline is "she votes, they vote, we vote" and the tagline is #shevotesboston. Besides having those taglines and the URL boston.gov/suffrage, we are open to most design ideas for this campaign, particularly illustration style designs. 

Would you be able to make us a couple of mockups to review and present to the digital billboard review group? 

Let us know what a realistic timeline is for this.

There are also other potential components of this campaign that we can discuss IRL.

**Comment by sebastianebarb** [comment]
![Suffrage project Voter Campaign #1468-03](https://user-images.githubusercontent.com/20642299/73455500-32f26080-433e-11ea-8371-49e242b64908.png)

**Comment by sebastianebarb** [comment]
https://docs.google.com/document/d/1tmFHb_-CYKY5qdQKyjCPERxVwH4RLZdUK33-FOuX9ug/edit?usp=sharing

**Comment by sebastianebarb** [comment]
![Suffrage project Voter Campaign #1468-03](https://user-images.githubusercontent.com/20642299/73847394-32514280-47f4-11ea-890e-ce95d70a88b1.png)

**Comment by sebastianebarb** [comment]
Hi Sebastian,

We shared this content with the Barbara Lee team today and here's the feedback we received:
-Is it possible to make variations with different types of people on each billboard? (while we loved the present vs historical idea, it seems like folks are having trouble understanding it without explanation)
-They are wondering if it is possible to have another older-looking person?
-Is it possible to have more people on each billboard?
Let us know your thoughts on the feasibility of these. 

Also, I have edited several of the taglines to be shorter/more concise. Updated versions here.

Thank you! Let us know if you want to discuss further. 

Best,
Allie
